js/repl-worker.js

```diff
@@ -34,6 +34,9 @@ onmessage = function(e) {
       postMessage({ type: 'error', data: buffer });
     }
   } catch (err) {
-    postMessage({ type: 'error', data: buffer });
+    postMessage({
```

Comment:
The buffer may actually be non empty, so rather append err.message to the buffer and then post the buffer.